apps/communications/dialer/test/marionette/keypad_test.js

```diff
@@ -7,7 +7,8 @@ var ReflowHelper =
 marionette('Dialer > Keypad', function() {
   var assert = require('assert');
 
-  var client = marionette.client({ profile: Dialer.config });
+  var client = marionette.client({ profile: Dialer.config,
+  desiredCapabilities: { raisesAccessibilityExceptions: false } });
```

Comment:
You got tricked the same way as I did in bug 1232680 :sweat_smile:  It took me a while to figure out that `desiredCapabilities` is not at the same level as `settings`. `settings` and `prefs` should be nested under `profile`.

See this example: https://github.com/mozilla-b2g/gaia/pull/33901/files#diff-ef4efacac7cfe3c475dc13f5c99492acR20